tests/caasp/stack_update.pm

```diff
@@ -45,11 +45,15 @@ sub orchestrate_velum_reboot {
         send_key_until_needlematch "velum-$n-nodes-outdated", 'f5', 10, 120;
     }
 
+    # Workaround for bsc#1099015
     if (check_screen 'velum-update-admin', 0) {
         record_soft_failure 'bsc#1099015 - Update admin node still visible after reboot';
-        sleep 60;
+        for (my $i = 1; $i <= 10; $i++) {
```

Comment:
`for (1..10)` would be cleaner since you are not using `$i` variable

Reply:
I wasn't aware of that syntax. thanks :+1: